uber/automated_emails.py

```diff
@@ -365,8 +365,6 @@ def __init__(self, conf):
          lambda a: a.paid == c.PAID_BY_GROUP and a.amount_extra and a.amount_paid == a.amount_extra,
          needs_approval=False)
 
-AutomatedEmail(Attendee, '{EVENT_NAME} payment refunded', 'reg_workflow/payment_refunded.txt',
```

Comment:
Separate bugfix -- this template doesn't exist so it spits out an error anytime someone tries to preview it.